src/cmd/cmd_replay.ml

```diff
@@ -79,6 +79,13 @@ let run_file ~unsafe ~optimize filename model =
         Fmt.epr "Got value %a but expected a f64 value." V.pp v;
         assert false
 
+    let symbol_range _ _ =
+      match model.(next ()) with
+      | V.I32 n -> n
+      | v ->
+        Fmt.epr "Got value %a but expected a i32 value." V.pp v;
+        assert false
+
```

Comment:
You also have to add the function below to the list of functions: https://github.com/OCamlPro/owi/pull/530/files#diff-fc3fa460100074e98bfc853fb3a78fd09509cc21162389c6f9374fc4c5eb8242R93-R130